cypress/e2e/awx/dashboard/welcomeModal.cy.tsx

```diff
@@ -4,14 +4,14 @@ describe('dashboard: Welcome modal', () => {
   });
 
   it('renders the Welcome Modal on the dashboard', () => {
-    cy.visit(`/ui_next/dashboard`);
+    cy.navigateTo(/^Dashboard$/);
```

Comment:
I had these links this way due to the pagination in EDA. I would like the links to stay how they are.

Reply:
We should not be using hard coded urls in tests.
It makes the code hard to refactor and keeps the routes from being configurable.
The tests should also not be relying on the querystring params as that might change.
EDA needs to support filtering on the table so that items can be found by tests.
If we want to hack that to work until there are over 100 items, then we can use cy.setTablePageSize